package exception;

public class Example3 {

	public static void main(String[] args) {
		
		String s = "Bangalore";
		
		try
		{
			char c = s.charAt(20);
			System.out.println(c); // n
		}
		catch (RuntimeException e) {
			// TODO: handle exception
		}
		
		try
		{
			int k = 10/0; // Arithmetic
			System.out.println(k);
		}
		catch (ArithmeticException e) {
			e.printStackTrace();
		}
		
		System.out.println("main method end...");
		
	}
}
